diff --git a/libiscsi-1.18.0/configure.ac b/libiscsi-1.18.0/configure.ac
index 1952e6d..de414a0 100644
--- a/libiscsi-1.18.0/configure.ac
+++ b/libiscsi-1.18.0/configure.ac
@@ -48,8 +48,7 @@ AM_CONDITIONAL(BUILD_MANPAGES,

 AC_CONFIG_HEADER(config.h)

-AC_CHECK_LIB([gcrypt], [gcry_control])
-AM_CONDITIONAL([HAVE_LIBGCRYPT], [test $ac_cv_lib_gcrypt_gcry_control = yes])
+AM_CONDITIONAL([HAVE_LIBGCRYPT], [test no = yes])

 AC_CACHE_CHECK([for sin_len in sock],libiscsi_cv_HAVE_SOCK_SIN_LEN,[
 AC_TRY_COMPILE([#include <sys/types.h>
@@ -106,6 +105,7 @@ AC_TRY_COMPILE([
 #include <rdma/rdma_verbs.h>],
 [int ibv = IBV_WC_SEND;],
 libiscsi_cv_HAVE_LINUX_ISER=yes,libiscsi_cv_HAVE_LINUX_ISER=no)])
+libiscsi_cv_HAVE_LINUX_ISER=no
 if test x"$libiscsi_cv_HAVE_LINUX_ISER" = x"yes"; then
     AC_DEFINE(HAVE_LINUX_ISER,1,[Whether we have iSER support])
 fi
@@ -143,6 +143,8 @@ AC_CHECK_MEMBER([struct CU_SuiteInfo.pSetUpFunc],
 AC_SEARCH_LIBS(clock_gettime, rt, [
 	       AC_DEFINE([HAVE_CLOCK_GETTIME],1,[Define if clock_gettime is available])])

+CFLAGS="$CFLAGS -pthread"
+LIBS="$LIBS -pthread"

 AC_CONFIG_FILES([Makefile]
 		[doc/Makefile]
